docs/source/how_to/how_to_algorithm_selection.ipynb

```diff
@@ -35,34 +63,92 @@
    "source": [
     "import numpy as np\n",
     "\n",
-    "import optimagic as om"
+    "import optimagic as om\n",
+    "\n",
+    "\n",
+    "def trid_scalar(x):\n",
+    "    \"\"\"Implement Trid function: https://www.sfu.ca/~ssurjano/trid.html.\"\"\"\n",
+    "    return ((x - 1) ** 2).sum() - (x[1:] * x[:-1]).sum()\n",
+    "\n",
+    "\n",
+    "def trid_gradient(x):\n",
+    "    \"\"\"Calculate gradient of trid function.\"\"\"\n",
+    "    l1 = np.insert(x, 0, 0)\n",
+    "    l1 = np.delete(l1, [-1])\n",
+    "    l2 = np.append(x, 0)\n",
+    "    l2 = np.delete(l2, [0])\n",
+    "    return 2 * (x - 1) - l1 - l2"
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": null,
+   "cell_type": "markdown",
    "metadata": {},
-   "outputs": [],
    "source": [
-    "@om.mark.least_squares\n",
-    "def sphere(params):\n",
-    "    return params\n",
+    "## Step 1: Theory\n",
+    "\n",
+    "The below decision tree offers a practical guide on how to narrow down the set of algorithms to experiment with, based on the theoretical properties of your problem:\n",
+    "\n",
+    "```{mermaid}\n",
+    "graph LR\n",
+    "    classDef highlight fill:#FF4500;\n",
+    "    A[\"Do you have<br/>nonlinear constraints?\"] -- yes --> B[\"differentiable?\"]\n",
+    "    B[\"differentiable?\"] -- yes --> C[\"'ipopt', 'nlopt_slsqp', 'scipy_trust_constr', ...\"]\n",
+    "    B[\"differentiable?\"] -- no --> D[\"'scipy_cobyla', 'nlopt_cobyla', ...\"]\n",
```

Comment:
The text in the terminal nodes is not rendered.

![image](https://github.com/user-attachments/assets/0199a867-e67f-4556-ab0e-932d4aec0231)

Reply:
It works locally. Any idea how to debug that?

Reply:
Is there a parameter to adjust the scaling? Or try top-down? There is not so much horizontal space in the rtd layout.